docs/mining/proof-of-stake.md

```diff
@@ -26,6 +26,30 @@ You will also need to [Obtain DCR](/getting-started/obtaining-dcr.md) if you hav
 
 ---
 
+## ** How Does It Work? ** ##
+
+Purchasing a ticket for PoS is quite simple (see below) but what happens to it after you buy it?
+A ticket will go through a few stages in its lifetime:
+
+1. You buy a ticket using your wallet.
+2. Tickets are mined into a block in order of their `ticketfee`. Note that the ticketfee is priced per KB. 
+A ticket transaction is about a third (solo) or half (pool) a KB so the actual fee you pay will be scaled accordingly.
+3. The ticket enters the 'mempool'. Only 20 tickets are mined into each block. This is where tickets wait to be mined.
```

Comment:
I'd swap 2 and 3, talk of entering mempool is always best before the ticket getting mined imo.